requirements.txt

```diff
@@ -2,6 +2,7 @@ torchvision>=0.5.0
 pretrainedmodels==0.7.4
 efficientnet-pytorch==0.7.1
 timm==0.6.13
+segment-anything-py
```

Comment:
pls fix the version, it is not a stable package

Reply:
I'd rather use the [original repo](https://github.com/facebookresearch/segment-anything) but it does not have a python package. If I pass a `git+https://github.com/facebookresearch/segment-anything` the `setup.py` fails. Do you want me just fix a version of the `segment-anything-py` or somehow point to the original repo?